changelog/unicode_properties_c.dd

```diff
@@ -0,0 +1,34 @@
+The Unicode property "C" aka "Other" has had the wrong properties associated with it.
+
+If you see this code (case insensitive), you should mitigate or fix your codebase.
+
+---
+unicode.c
+---
+
+or:
+
+---
+unicode.Other
+---
```

Comment:
```suggestion
If you use `unicode.c` or `unicode.Other` (case insensitive) from `std.uni`, you should mitigate or fix your codebase.
```

I would keep them inline, or make the code blocks actually compilable. `unicode.c` on its own reads like a filename to me.

Reply:
Cheers, I didn't know how to do that.